packages/database/src/TupaiaDatabase.js

```diff
@@ -276,9 +276,9 @@ export class TupaiaDatabase {
     const sql = `
      with recursive findParents as (
        select * from ${recordType}
-         where ${idKey} = '${id}'
+         where ${idKey} in ('${Array.isArray(id) ? id.join("','") : id}')
        union
-         select ${recordType}.* from ${recordType}
+         select distinct(${recordType}.*) from ${recordType}
```

Comment:
Added a `distinct` here because searching multiple children can result in multiples of the same parent